udata/core/discussions/csv.py

```diff
@@ -0,0 +1,21 @@
+from udata.frontend import csv
+
+from .models import Discussion
+
+
+@csv.adapter(Discussion)
+class DiscussionCsvAdapter(csv.Adapter):
+    fields = (
+        "id",
+        "user",
+        "subject",
+        ("subject_class", "subject.class"),
+        ("subject_id", "subject.id"),
+        "title",
+        ("size", lambda o: len(o.discussion)),
+        ("participants", lambda o: ",".join(set(msg.posted_by.id for msg in o.discussion))),

```

Comment:
I think ObjectId needs to be casted to str
```suggestion
        ("participants", lambda o: ",".join(set(str(msg.posted_by.id) for msg in o.discussion))),

```